turbopack/crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -696,7 +696,6 @@ impl FileSystem for DiskFileSystem {
 
     #[turbo_tasks::function(fs)]
     async fn write(&self, fs_path: Vc<FileSystemPath>, content: Vc<FileContent>) -> Result<()> {
-        mark_session_dependent();
```

Comment:
It might be good to add comments here. Otherwise I could see somebody thinking this code is wrong and adding `mark_session_dependent()` back later.